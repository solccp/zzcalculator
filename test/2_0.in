54
74
C          -2.46910700     2.84886900     0.00000000
C          -3.70959000     3.55861100     0.00000000
C          -3.67473400     4.99512300     0.00000000
C          -2.48853300     5.67998500     0.00000000
C          -1.22703900     4.99191500     0.00000000
C          -1.23262700     3.56275900     0.00000000
C          -4.92074200     2.84099300     0.00000000
C          -2.47015000     1.42615600     0.00000000
C          -3.70173700     0.71388100     0.00000000
C          -4.93663100     1.43330700     0.00000000
C           0.00000000     2.85228800     0.00000000
C           0.00000000     1.42244300     0.00000000
C          -1.23186600     0.71121300     0.00000000
C           0.00000000     5.68201700     0.00000000
C           1.22703900     4.99191500     0.00000000
C           1.23262700     3.56275900     0.00000000
C           2.48853300     5.67998500     0.00000000
C           2.46910700     2.84886900     0.00000000
C           2.47015000     1.42615600     0.00000000
C           1.23186600     0.71121300     0.00000000
C           3.67473400     4.99512300     0.00000000
C           3.70959000     3.55861100     0.00000000
C           1.23186600    -0.71121300     0.00000000
C           0.00000000    -1.42244300     0.00000000
C          -1.23186600    -0.71121300     0.00000000
C          -2.47015000    -1.42615600     0.00000000
C          -3.70173700    -0.71388100     0.00000000
C          -4.93663100    -1.43330700     0.00000000
C          -6.16326300    -0.68485000     0.00000000
C          -6.16326300     0.68485000     0.00000000
C           3.70173700     0.71388100     0.00000000
C           3.70173700    -0.71388100     0.00000000
C           2.47015000    -1.42615600     0.00000000
C           2.46910700    -2.84886900     0.00000000
C           1.23262700    -3.56275900     0.00000000
C           0.00000000    -2.85228800     0.00000000
C           4.92074200     2.84099300     0.00000000
C           4.93663100     1.43330700     0.00000000
C          -4.92074200    -2.84099300     0.00000000
C          -2.46910700    -2.84886900     0.00000000
C          -3.70959000    -3.55861100     0.00000000
C          -1.23262700    -3.56275900     0.00000000
C           6.16326300     0.68485000     0.00000000
C           6.16326300    -0.68485000     0.00000000
C           4.93663100    -1.43330700     0.00000000
C           4.92074200    -2.84099300     0.00000000
C           3.70959000    -3.55861100     0.00000000
C           3.67473400    -4.99512300     0.00000000
C           2.48853300    -5.67998500     0.00000000
C           1.22703900    -4.99191500     0.00000000
C          -3.67473400    -4.99512300     0.00000000
C          -1.22703900    -4.99191500     0.00000000
C          -2.48853300    -5.67998500     0.00000000
C           0.00000000    -5.68201700     0.00000000
