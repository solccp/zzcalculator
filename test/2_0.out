 980 + 3308 x + 4458 x^2 + 3065 x^3 + 1140 x^4 + 225 x^5 + 22 x^6 + 1 x^7
 total: 13199
